### DISCUSSION TITLE
Multiple Islands

### DISCUSSION THREAD

**Discussion by DeOpping** [BG-Software-LLC/SuperiorSkyblock2#859] [Pending Feature Requests]
Hello, I am wanting to make this into a prison cells plugin, rather than use plots. A lot of people have done so for their prison servers, and it is quite easy, but there are a few things lacking in here that are in plots..

For instance, with plots, you can have your own, multiple even, while being trusted on other plots, even if you disconnect..

So, really, what I'm asking is..
1) The ability to allow players more than 1 island, or the ability to have an island and join another player's island, preferably both?
2) The ability to not remove members from a coop when they disconnect..
3) A way to teleport to islands you're a coop member of, or a way to teleport to islands you're a member of.

Basically, in PlotSquared terms.. /plotme trust, /plotme claim, /plotme visit
I'm sure a lot of people might not see the practicality of this, so, if there's a way to do it through the API, I will gladly do it myself, or I can try editing the src, but I probably have a bit more to learn if I am going to do that..
